src/lang/pt-br.json

```diff
@@ -184,25 +189,25 @@
 	"MOVEMENT": "Movimento",
 	"MODIFIERS": "Modificadores",
 	"NOTIFY.YOU_ARE_BROKEN": "Você está acabado!",
-	"NOTIFY.YOUR_ITEM_BROKE": "Seu equipamento está quebrado!",
```

Comment:
The book translated gear as "equipamento", mas also uses item on occasion. Here I tried to remain consistent with the English version and since that can be applied to gear or weapon, item is more generic.